resources/conf/rate-limiter-config.yaml

```diff
@@ -0,0 +1,27 @@
+# Sample config until, Adapter xDS implementation is done
+domain: default
+descriptors:
+  - key: org
+    value: wso2
+    descriptors:
+      - key: vhost
+        value: localhost
+        descriptors:
+          # API level policy
+          - key: path
+            descriptors:
+              - key: method
+                descriptors:
+                  - key: policy
+                    value: $_5PerMin
+                    descriptors:
+                      - key: condition
+                        value: default
+                        rate_limit:
+                          unit: minute
+                          requests_per_unit: 5
+                      - key: condition
+                        value: c1
+                        rate_limit:
+                          unit: minute
+                          requests_per_unit: 3
```

Comment:
Do we need this condition section (since we are not supporting conditions ATM) ?

Reply:
yeah let's update these changes with your changes, this is done with your PR changes before we remove the conditional policy.